selenium/src/test/java/au/org/emii/portal/tests/step1/FacetDateGoButton.java

```diff
@@ -0,0 +1,38 @@
+package au.org.emii.portal.tests.step1;
+
+import au.org.emii.portal.tests.BaseTest;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class FacetDateGoButton extends BaseTest {
+
+    private static Logger log = Logger.getLogger(FacetDateGoButton.class.getName());
+
+    @Test
+    public void goButtonTest() {
+        log.info("Loading search page - Step 1");
+        getDriver().get(AODN_PORTAL_SEARCH_PAGE);
+
+        log.info("Selecting Date Facet");
+        portalUtil.selectFacetHeading("Parameter");
+        portalUtil.selectFacetHeading("Platform");
+        portalUtil.selectFacetHeading("Date (UTC)");
+
+        WebElement goButton = webElementUtil.findElement(By.xpath("//button[contains(.,'Go')]"));
+        WebElement parentElement = goButton.findElement(By.xpath("./../../../../.."));
+        String cssClass = parentElement.getAttribute("class");
+        Assert.assertTrue(cssClass.contains("x-item-disabled"), "Go button should not be enabled");
```

Comment:
@bhasin85 I have an issue with this as there are three buttons on the page that match the `"//button[contains(.,'Go')]` "regex. 
I think its just luck that this works, and that is why my previous commit had such an involved regex

This is what i get in Chromes console
```
$x('//button[contains(.,"Go")]')
Array[3]0: button#ext-gen181.x-btn-text1: button#ext-gen188.x-btn-text2: button#ext-gen192.x-btn-textlength: 3__proto__: Array[0]
```